angr/knowledge_plugins/xrefs/xref.py

```diff
@@ -34,9 +34,9 @@ def type_string(self):
 
     def __repr__(self):
         return "<XRef %s: %s->%s>" % (
-                XRefType.to_string(self.type),
+                self.type_string,
                 "%#x" % self.ins_addr if self.ins_addr is not None else "%#x[%d]" % (self.block_addr, self.stmt_idx),
-                "%#x" % (self.dst if self.dst is not None else self.memory_data.addr)
+                "%#s" % self.dst if self.dst is not None else "%#x" % (self.memory_data.addr)
```

Comment:
What's the difference between `%#s` and `%s`?

Reply:
Nothing. `#`s alternate form isn't specified for strings. I missed it.